zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala

```diff
@@ -469,24 +466,28 @@ private[consumer] final class Runloop private (
    *   - Poll periodically when we are subscribed but do not have assigned streams yet. This happens after
    *     initialization and rebalancing
    */
-  def run: ZIO[Scope, Throwable, Any] =
+  def run: ZIO[Scope, Throwable, Any] = {
+    import Runloop.StreamOps
+
     ZStream
       .fromQueue(commandQueue)
       .timeoutFail[Throwable](RunloopTimeout)(runloopTimeout)
-      .takeWhile(_ != StopRunloop)
+      .takeWhile(_ ne RunloopCommand.StopRunloop)
```

Comment:
In general `eq` and `ne` should only be used when reference equality is essential. That is not the case here. To validate its correctness you have to lookup what `StopRunLoop` is (it is an object, ah, okay, no problem here...).

```suggestion
      .takeWhile(_ != RunloopCommand.StopRunloop)
```

Reply:
I'm sorry but I'm not sure to understand your comment 🤔

I'd prefer to keep `ne`. It checks less things than `!=` so is faster.
Scalac complains if you misuse `ne` or `eq`

Reply:
If StopRunLoop becomes a variable, reference equality may no longer be correct. One really should not use eq/ne frivolously. It will prevent problems later.

The performance gain is picoseconds at best. Compared to all the other things we do it is not worth talking about.